### ISSUE TITLE
Issues with encryption and command line compatibility (?)

### ISSUE THREAD

**Issue by guice** [Ne-Lexa/php-zip#1]
I'm attempting to create an encrypted zip using the traditional method:

````php
$zip->addFromFile($csv_path, pathinfo($csv_path, PATHINFO_BASENAME));
$zip->setPassword('tod', ZipEntry::ENCRYPTION_METHOD_TRADITIONAL);
$zip->saveAsFile($zip_name);
````

Unfortunately, when I try to unzip it using OSX Terminal, or CentOS command line, the password isn't getting accepted:

````
[guice@system ~]$ unzip testzip.zip 
Archive:  testzip.zip
[testzip.zip] test.csv password: 
password incorrect--reenter: 
password incorrect--reenter: 
   skipping: testzip.csv  incorrect password
````

Zip info:

````
[guice@system ~]$ unzip -v
UnZip 6.00 of 20 April 2009, by Info-ZIP.  Maintained by C. Spieler.  Send
bug reports using http://www.info-zip.org/zip-bug.html; see README for details.

Latest sources and executables are at ftp://ftp.info-zip.org/pub/infozip/ ;
see ftp://ftp.info-zip.org/pub/infozip/UnZip.html for other sites.

Compiled with gcc 4.4.7 20120313 (Red Hat 4.4.7-16) for Unix (Linux ELF) on Dec 15 2015.

UnZip special compilation options:
        COPYRIGHT_CLEAN (PKZIP 0.9x unreducing method not supported)
        SET_DIR_ATTRIB
        SYMLINKS (symbolic links supported, if RTL and file system permit)
        TIMESTAMP
        UNIXBACKUP
        USE_EF_UT_TIME
        USE_UNSHRINK (PKZIP/Zip 1.x unshrinking method supported)
        USE_DEFLATE64 (PKZIP 4.x Deflate64(tm) supported)
        UNICODE_SUPPORT [wide-chars, char coding: UTF-8] (handle UTF-8 paths)
        MBCS-support (multibyte character support, MB_CUR_MAX = 6)
        LARGE_FILE_SUPPORT (large files over 2 GiB supported)
        ZIP64_SUPPORT (archives using Zip64 for large files supported)
        VMS_TEXT_CONV
        [decryption, version 2.11 of 05 Jan 2007]

UnZip and ZipInfo environment options:
           UNZIP:  [none]
        UNZIPOPT:  [none]
         ZIPINFO:  [none]
      ZIPINFOOPT:  [none]
````

**Comment by Ne-Lexa** [comment]
Hello, @guice
Fixed in version 39f861633668c5472fb680dd03a014ee0e78b796 2.1.1
Upgrade version and write about new results.

**Comment by guice** [comment]
It works! Thanks. 👍
